web/src/pages/news/articles/index.page.tsx

```diff
@@ -1,11 +1,144 @@
+import { NewsList } from "@/components/Storyblok/News/NewsList/NewsList";
+import { NewsListNav } from "@/components/Storyblok/News/NewsListNav/NewsListNav";
+import { NewsListPagination } from "@/components/Storyblok/News/NewsListPagination/NewsListPagination";
+import { logger } from "@/logger";
+import { NewsStory } from "@/types/News";
+import { fetchStories, getStoryVersionFromQuery } from "@/utils/storyblok";
+
+import type { GetServerSidePropsContext } from "next";
+
+export type NewsArticlesProps = {
+	featuredStory: StoryblokStory<NewsStory>;
+	stories: StoryblokStory<NewsStory>[];
+	totalResults: number;
+	currentPage: number;
+	resultsPerPage: number;
+};
+
+const destinations = [
+	{ url: "/news/", title: "News" },
+	{ url: "/news/articles", title: "News articles" },
+	{ url: "/news/in-depth", title: "In-depth" },
+	{ url: "/news/blogs", title: "Blogs" },
+	{ url: "/news/podcasts", title: "Podcasts" },
+];
+
+export const ArticlesIndexPage = ({
+	stories,
+	currentPage,
+	totalResults,
+	resultsPerPage,
+	featuredStory,
+}: NewsArticlesProps): React.ReactElement => {
 	return (
 		<>
-			<h1>Articles Index Page</h1>
+			<NextSeo
+				title="News Articles"
+				openGraph={{ title: "News Articles" }}
+			></NextSeo>
+			<PageHeader
+				heading="News articles"
+				variant="fullWidthDark"
+				breadcrumbs={
+					<Breadcrumbs>
+						<Breadcrumb to="https://www.nice.org.uk/">Home</Breadcrumb>
+						<Breadcrumb to="https://www.nice.org.uk/news">News</Breadcrumb>
+						<Breadcrumb>Articles</Breadcrumb>
+					</Breadcrumbs>
+				}
+			/>
+			<NewsListNav destinations={destinations} />
+			<FeaturedStory story={featuredStory} />
+			<NewsList news={stories} />
+			<ActionBanner
+				title="Sign up for our newsletters and alerts"
+				cta={
+					<Button variant="cta" to="/news/nice-newsletters-and-alerts">
+						Sign up for newsletters and alerts
+					</Button>
+				}
+			>
+				Keeping you up to date with important developments at NICE
+			</ActionBanner>
+			<NewsListPagination
+				configuration={{
+					currentPage,
+					totalResults,
+					resultsPerPage,
+				}}
+			/>
 		</>
 	);
 };
 
+export async function getServerSideProps({ query }: GetServerSidePropsContext) {
+	logger.info("Start server side props for /news/articles list page");
+
+	const version = getStoryVersionFromQuery(query);
+	const page = Number(query.page) || 1;
+	const resultsPerPage = 5;
+
+	const storiesResult = await fetchStories(version, {
+		starts_with: "news/articles/",
+		per_page: resultsPerPage,
+		page,
+		sort_by: "content.date:desc",
+	});
+
+	const latestStoryResult = await fetchStories(version, {
+		starts_with: "news/articles/",
+		per_page: 1,
+		page: 1,
+		sort_by: "content.date:desc",
+	});
+
+	logger.info("Finish server side props for news articles list page");
+
+	if (!storiesResult || storiesResult.total === undefined) {
+		logger.error("Failed to fetch stories");
+		return { notFound: true };
```

Comment:
Thinking about it, I think we need to suss out how we want to handle pages where we are getting multiple stories but get an empty result set. In this case, returning `notFound: true` will redirect us to the 404 page, which technically isn't correct as this page doesn't exist; it just wouldn't be returning any items, either because Storyblok has gone down or somebody has wiped the news stories or whatever.

Maybe something to mob on, but I think the sensible thing for us to do might be just to render a "no stories found" message on the page or similar.

Reply:
This still needs sorting, sorry. The problem here is that if/when the Storyblok API falls over (which it inevitably will at some point) or if someone in content erases all the stories by accident and we get nothing back then this line will cause the page to return a 404. That's not correct - the page does exist, it just won't be showing any stories - and it'll have a negative effect on SEO.

In this case I think we need to show a simple error message - just something like "there are no stories to display" - and log an error (using `logger.error()`) so that we can find when it occurred in the logs.

Reply:
Ok sorted this.